pyproject.toml

```diff
@@ -9,7 +9,7 @@ license = "MIT"
 python = "^3.8,<3.11"
 
 [tool.poetry.group.dev.dependencies]
-demisto-sdk = "1.8.2"
+demisto-sdk = {git = "https://github.com/demisto/demisto-sdk.git", rev = "support_native_docker_image_in_lint"}
```

Comment:
please use here a commit sha not a branch name

Reply:
Still working on the sdk branch I'm 'looking' at, that's why I chose the branch name.